Linux/Car_Client/chronoscomm.cpp

```diff
@@ -20,6 +20,8 @@
 #include <cmath>
 
 #define PIN_OUT 4
+#define HEARTBEAT_TIME_MS 1000
```

Comment:
Is this a timeout? Perhaps a name such as MAX_HEARTBEAT_PERIOD_MS or similar might be clearer?